jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/metrics/SimpleMetricsProvider.java

```diff
@@ -33,7 +34,8 @@ public MeterRegistry getMeterRegistry() {
 
     @Override
     public void scrape(HttpAction action) {
-        ServletOps.errorNotImplemented( "SimpleMeterRegistry isn't scrapeable" );
+        ServletOps.responseSendError(action.response, HttpSC.NOT_IMPLEMENTED_501,
+                "SimpleMeterRegistry isn't scrapeable" );
```

Comment:
Something strange is going on.
`ServletOps.errorNotImplemented` should do that via throwing `ActionErrorException` which is caught in `ActionBase`. Ah - `ActionMetrics.doGet` is doing it's own thing not using the common lifecycle.

Using `ServletOps.errorNotImplemented` and the `ActionMetrics` below for the common execution c. Is there anything in the lifecycle that was a problem?

```
public class ActionMetrics extends ActionCtl {
    
    public ActionMetrics() { }

    @Override
    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
        super.doCommon(req, resp);
    }

    @Override
    protected void perform(HttpAction action) {
        MetricsProviderRegistry.get().scrape( action );
    }
}
```

Reply:
Yeah that works great - I've just pushed the change.

The previous implementation was left over from when I was doing that `/$/extras/*` stuff that's gone now